packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade.constants.js

```diff
@@ -0,0 +1,8 @@
+export const ISSUE_TYPE_ID = 38089;
+export const TICKET_BASE_URL =
+  'https://www.ovh.com/manager/dedicated/index.html#/support/tickets';
```

Comment:
Maybe we should use the `RedirectionService` (and we'll do a batch replace when switching solution)  or @cbourgois work on URLs as this won't work for CA and US for instance

Reply:
We should use `url-builder` (cf. https://projects.dsi.ovh/browse/DTRSD-25255) 
If a feature is not available for a region, it should use feature flipping.

```js
import { buildURL } from '@ovh-ux/ufrontend/url-builder';

const ticketsURL = buildURL('dedicated', '#/support/tickets');
```

But in your case you should avoid to build URL yourself using concatenation, use something like (in `packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.controller.js`) : 

```js
import { buildURL } from '@ovh-ux/ufrontend/url-builder';

const ticketURL = buildURL('dedicated', '#/support/tickets/:ticketId', { ticketId });
```

If you have to restrict the URL for a specific region, use `Environment.getRegion()` from `@ovh-ux/manager-config`.

Reply:
thanks, it's done now :)

Reply:
@Heiiji 
You are using `RedirectionService` and not `url-builder` (https://github.com/ovh/manager/pull/4188/files#diff-81bb9799be0607aea4500d81c0504e796dfa35a2a25aa08b1158df3c6a0bd2a6R61-R63)
```js
ticketUrl: `${this.RedirectionService.getURL('ticket', {
  ticketId,
})}`,
```
Any reason ?

Reply:
yes, i've done it before your comm and i change it now. github isn't real time platform :o